src/targets/kotlin/partials/nested.hbs

```diff
@@ -0,0 +1,16 @@
+data class {{{upperFirst name}}}(
```

Comment:
The `{{name}}` field should be processed the same way that the parent schema of this nested field is processed. if it isn't, it is possible that they will not be the same, and in languages where classes are not title case (ew) this won't work.